### ISSUE TITLE
Update Toolkit page's mobile filter

### ISSUE THREAD

**Issue by daniellex0** [hackforla/website#1329]
Dependency
~#2171~

### Overview
We need to redesign the [Toolkit page](https://www.hackforla.org/toolkit/) mobile filter banner, because the filter categories' font is too small when all filters are displayed in one row in mobile

### Action Items

Design

- [x] Review the [Toolkit page](https://www.hackforla.org/toolkit/) filter in mobile (using Inspect Element)
- [x] View the [Homepage](https://www.hackforla.org/#) filter in mobile for an example of effectively adapting filter to mobile
- [x] Create a rough recreation of the current Toolkit page mobile filter in the [Toolkit page file on Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=1548%3A1967) 
- [x] Redesign the Toolkit page mobile filter
- [x] Get design review from UX team

Development
- [x] Review the mobile (dropdown) version of the Toolkit page filter banner on [Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=1548%3A1967) (in pink rectangle)
- [ ] Develop the **mobile-only** dropdown filter (should stay same beige banner in tablet and desktop - the change should only be in mobile)
   - [ ] Follow same code as dropdown in [Wins page](https://www.hackforla.org/wins/)

### Resources/Instructions
[Toolkit page](https://www.hackforla.org/toolkit/)
[Figma Toolkit page file](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=1548%3A1967) (final mobile version in pink rectangle]
[Homepage](https://www.hackforla.org/#)

New mobile version of Toolkit page filter (beige banner):

<img width="245" alt="Screen Shot 2021-04-27 at 9 21 29 AM" src="https://user-images.githubusercontent.com/67400366/116277080-fc221a80-a739-11eb-87da-6951a9066340.png">

**Comment by daniellex0** [comment]
@shinhope will figure out improvements to desktop version as well

**Comment by shinhope** [comment]
mobile version ideas:
![image](https://user-images.githubusercontent.com/79497980/112912638-db1fc880-90ac-11eb-97f5-fa1c200e7340.png)

**Comment by shinhope** [comment]
new desktop versions:
![image (1)](https://user-images.githubusercontent.com/79497980/112912665-ea067b00-90ac-11eb-80c3-12914792da06.png)
![image (2)](https://user-images.githubusercontent.com/79497980/112912669-ed016b80-90ac-11eb-8801-a300f6ba187e.png)

**Comment by akibrhast** [comment]
@daniellex0 What is the news on this, where do you want to go with this?

**Comment by daniellex0** [comment]
@akibrhast We are discussing, will add update here when it's finalized!

**Comment by daniellex0** [comment]
New final mobile dropdown version of filter:

<img width="245" alt="Screen Shot 2021-04-27 at 9 21 29 AM" src="https://user-images.githubusercontent.com/67400366/116277147-0d6b2700-a73a-11eb-8fdf-b5d86cf1e79c.png">

[See design on Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=1548%3A1967) (in pink rectangle)

**Comment by qiqicodes** [comment]
@danyc23 Can you let us know your progress in regards to this issue? Thank you =]

Progress:
Blockers:

**Comment by danyc23** [comment]
Blockers:  Haven't find any blockers yet.
Progress: Working on it.
ETA: May 19, 2021

**Comment by wendywilhelm10** [comment]
@danyc23 Could you please update your issue?

**Comment by danyc23** [comment]
Blockers: No blockers.
Progress: Still working on it.
ETA: June 2, 2021

**Comment by Sihemgourou** [comment]
@danyc23 
Please add update
1. Progress
2. Blockers
3. Availability
4. ETA

**Comment by Sihemgourou** [comment]
@danyc23 has been missing for more than 2 weeks.

**Comment by ExperimentsInHonesty** [comment]
@hackforla/website-merge can I get your thoughts on this... each guide has a list of categories inside of it and it might display on multiple categories so if you have the ability to check several boxes then it sounds technically more complicated than just allowing only one category to be selected.  Which seems like it would call for radio selector or just the name selector.  Please share thoughts.

This is an example of a guide.  This one only has one category, but you get the idea
https://github.com/hackforla/website/blob/gh-pages/_guide-pages/2FA.html

**Comment by mgodoy2023** [comment]
Whenever this gets resolved please let me know because I'd love to pick it up. Thanks!

**Comment by ExperimentsInHonesty** [comment]
@danyc23 I see that you reassigned this to yourself, but I'll need more info that that, because we didn't have an update from Since May 28th, even though we asked a couple of times.  So please provide an update now so I can decide if we can keep you on this issue.  Apologies if you notified the team that you were going to be out and I am just unaware.  For reference though, you do have to put that information on your issue or ask someone else to do that.  Then based on the projected absence we decide to keep you on the issue or put it back in the prioritized backlog.

**Comment by danyc23** [comment]
> @danyc23 I see that you reassigned this to yourself, but I'll need more info that that, because we didn't have an update from Since May 28th, even though we asked a couple of times. So please provide an update now so I can decide if we can keep you on this issue. Apologies if you notified the team that you were going to be out and I am just unaware. For reference though, you do have to put that information on your issue or ask someone else to do that. Then based on the projected absence we decide to keep you on the issue or put it back in the prioritized backlog.

Hi Bonnie, I was away for personal reasons and I mentioned it to Josh. I have done some advancements to this issue and still need to fix some small things to finish the filtering functionality. Attached you will see an image of my progress. Let me know if you have any question 

<img width="426" alt="Screen Shot 2021-07-20 at 2 09 52 PM" src="https://user-images.githubusercontent.com/42159475/126396309-e9970b4b-ccea-4bab-a5f1-94cc8448bb57.png">

**Comment by ExperimentsInHonesty** [comment]
@danyc23 - I want to talk again to the design team.  I'm not satisfied with the solution they have proposed.

**Comment by danyc23** [comment]
> @danyc23 - I want to talk again to the design team. I'm not satisfied with the solution they have proposed.

Ok, I'll be waiting for the new design. Should I move it back to In Review?

**Comment by Aveline-art** [comment]
@danyc23 There has been no updates on this, so this issue is being recycled. When you are ready to join us, please come to a meeting so that we can fill you in on our new procedures and get you started with us!

**Comment by blakes24** [comment]
@hackforla/website-design I have a few questions about the design.

1. The non-mobile view of this page does not have the option to filter for multiple categories at once, but the proposed mobile view does (checkboxes rather than standard dropdown). I just want to make sure this is correct because it seems like the page should have the same functionality no matter the screen size.

2. The design for the filter includes a checkbox for the option "all". The other pages on the site that use checkbox dropdowns default to all categories if nothing is selected and do not have a checkbox for all. Should the "all" option be removed to remain consistent with the rest of the site?

3. The design shows the top bar of the dropdown displaying the category that is selected. This makes sense for a traditional dropdown where only one category can be selected at a time. How should this be handled when multiple categories are selected?

**Comment by ExperimentsInHonesty** [comment]
@blakes24 sorry about the confusion, the is issue need a new design before we can make the change.  I added a dependency

**Comment by ExperimentsInHonesty** [comment]
@blakes24 we got the design done, so now the issue will go into the prioritized backlog.

**Comment by ExperimentsInHonesty** [comment]
- @blakes24 opps.  Adam already made an issue for implementing.  its #2391  I will close this one.  Just letting you know since you had been assigned. The new issue is in the backlog.
